### PR TITLE
Add proper development versioning

### PR DESCRIPTION
**Pull request by corranwebster**
Update the versioning system in the same way as Traits, Pyface, etc.

### CODE REVIEW

**Comment by corranwebster** [comment]
@itziakos Could you please review.

File: .travis.yml [hunk 1]
```diff
@@ -12,8 +12,9 @@ install:
   - pip install unittest2
   - pip install pygments
   - sudo apt-get install python-numpy
-  # Test against the current master of traits, traitsui and enable
+  # Test against the current master of traits and pyface
   - pip install git+http://github.com/enthought/traits.git#egg=traits
+  - pip install git+http://github.com/enthought/pyface.git#egg=pyface
```
Comment (itziakos) [inline .travis.yml]: it might be better to add these dependences into a `requirements.txt` file.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,15 @@
+include CHANGES.txt
+include LICENSE.txt
+include MANIFEST.in
```
Comment (itziakos) [inline MANIFEST.in]: you do not need to include this

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+include README.rst
+include image_LICENSE.txt
+include image_LICENSE_Eclipse.txt
+include image_LICENSE_Nuvola.txt
+include TODO.txt
+graft docs
+prune docs/build
+recursive-exclude docs *.pyc
+graft examples
+recursive-exclude examples *.pyc
```
Comment (itziakos) [inline MANIFEST.in]: I think that these are excluded by default

Comment (itziakos) [inline MANIFEST.in]: I see you graft the directory above

File: MANIFEST.in [hunk 4]
```diff
@@ -0,0 +1,15 @@
+include CHANGES.txt
+include LICENSE.txt
+include MANIFEST.in
+include README.rst
+include image_LICENSE.txt
+include image_LICENSE_Eclipse.txt
+include image_LICENSE_Nuvola.txt
+include TODO.txt
+graft docs
```
Comment (itziakos) [inline MANIFEST.in]: So we are actually including the pdf files?

**Comment by itziakos** [comment]
it looks ok to me althought, a little verbose.

File: MANIFEST.in (hunk 4 above)
Comment (corranwebster) [inline MANIFEST.in]: Yes.

Whether or not they should be there still (after 7 years!) when the same content is available online is another question.  But not one for this PR.

File: MANIFEST.in (hunk 3 above)
Comment (corranwebster) [inline MANIFEST.in]: Yep, I think you have to do this - graft gives everything.

File: MANIFEST.in (hunk 2 above)
Comment (corranwebster) [inline MANIFEST.in]: I copied this from Mark's traits version.  I'll leave it in for now.

**Pull request merged**